lib/src/commands.rs

```diff
@@ -556,6 +555,7 @@ impl Command for SendCommand {
                 Ok(amt) => amt,
                 Err(e)  => {
                     if args[1] == "entire-verified-zbalance" {
+                        let fee = lightwallet::fee::get_default_fee(lightclient.wallet.read().unwrap().last_scanned_height());
                         match lightclient.wallet.read().unwrap().verified_zbalance(None).checked_sub(fee) {
                             Some(amt) => amt,
                             None => { return format!("Not enough in wallet to pay transaction fee") }
```

Comment:
perhaps this can be a bit more descriptive. Something like:
```suggestion
                            None => { return format!("Not enough in wallet to pay transaction fee of {}", fee) }
```